src/shogun/io/SGIO.cpp

```diff
@@ -146,7 +146,7 @@ class SGIO::RedirectSink : public spdlog::sinks::base_sink<std::mutex>
 	std::shared_ptr<spdlog::sinks::sink> stderr_sink;
 };
 
-SGIO::SGIO() : show_progress(false)
+SGIO::SGIO() : show_progress(false), syntax_highlight(false)
```

Comment:
@vigsterkr valgrind was complaining about this. I am assuming by default this should be `false`?

Reply:
👍